src/us/kbase/narrativejobservice/NarrativeJobServiceServer.java

```diff
@@ -97,6 +97,8 @@
     public static final String CFG_PROP_MONGO_USER_UJS = "ujs-mongodb-user";
     public static final String CFG_PROP_MONGO_PWD_UJS = "ujs-mongodb-pwd";
 
+    public static final String CFG_PROP_CONDOR_JOB_DATA_DIR= "condor-submit-workdir";
```

Comment:
Already merged in 121.. Will make branches based off of develop, instead of other branches from now on.